common/alert/index.json-ld

```diff
@@ -0,0 +1,35 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Alert",
+		"fr": "Alertes"
+	},
+	"description": {
+		"en": "Alerts with custom styles.",
+		"fr": "Alertes avec des styles personalisés."
+	},
+	"modified": "2023-03-03",
+	"componentName": "tables",
```

Comment:
This component name is "alert". 
```suggestion
	"componentName": "alert",
```

Note: Having the wrong component name create a broken link in our main index page when it get generated by jekyll.